forge-gui/res/cardsfolder/upcoming/evolving_adaptive.txt

```diff
@@ -0,0 +1,13 @@
+Name:Evolving Adaptive
+ManaCost:G
+Types:Creature Phyrexian Warrior
+PT:0/0
+K:etbCounter:OIL:1
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ X | AddToughness$ X | Description$ CARDNAME gets +1/+1 for each oil counter on it.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+powerGTY+Other,Creature.YouCtrl+toughnessGTZ+Other | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever another creature enters the battlefield under your control, if that creature has greater power or toughness than CARDNAME, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | CounterType$ OIL
```

Comment:
intervening if not handled

Reply:
Should this always trigger even if power and toughness is less when it etb's ?

Reply:
> If the ability triggers, the stat comparison will happen again when the ability tries to resolve. If neither stat of the new creature is greater, the ability will do nothing. If the creature that entered the battlefield leaves the battlefield before the ability tries to resolve, use its last known power and toughness to compare the stats.

Reply:
Ah I see, thanks for the heads up

Reply:
Like this ?

Reply:
No reason to change the trigger event

you can read up on the rule here:
https://yawgatog.com/resources/magic-rules/#R6034

Reply:
Ah It checks 2 times

Reply:
getting there, but if you read closely the YouCtrl is actually not part of the condition (belongs only to the trigger event)